### ISSUE TITLE
yarn install error

### ISSUE THREAD

**Issue by siriwatknp** [emotion-js/emotion#2414]
**Current behavior:**

got this error after clone emotion and then `yarn`

```
error /Users/siriwatknp/Personal-Repos/emotion/node_modules/sharp: Command failed.
Exit code: 1
Command: (node install/libvips && node install/dll-copy && prebuild-install) || (node-gyp rebuild && node install/dll-copy)
Arguments: 
Directory: /Users/siriwatknp/Personal-Repos/emotion/node_modules/sharp
Output:
info sharp Downloading https://github.com/lovell/sharp-libvips/releases/download/v8.7.4/libvips-8.7.4-darwin-arm64v8.tar.gz
/Users/siriwatknp/Personal-Repos/emotion/node_modules/sharp/install/libvips.js:86
          throw new Error(`Status ${response.statusCode}`);
          ^

Error: Status 404
    at /Users/siriwatknp/Personal-Repos/emotion/node_modules/sharp/install/libvips.js:86:17
```

I checked https://github.com/lovell/sharp-libvips/releases/download/v8.7.4/libvips-8.7.4-darwin-arm64v8.tar.gz got 404. Not sure if I missed any step.

**To reproduce:**

1. clone the project
2. `yarn` at latest commit `dcc72d06ace804330fd285a76c8574f3a89001f9`
3. 

**Expected behavior:**

should be able to complete installing without error

**Environment information:**

tried with node v14.17.0 and 16.3.0

**Comment by iChenLei** [comment]
It look like not `emotion`'s issue, maybe you should report bug to `https://github.com/lovell/sharp-libvips` ?

**Comment by sarayourfriend** [comment]
> tried with node v14.17.0 and 16.3.0

This is likely the issue. I had a similar problem but the `.nvmrc` for the repository indicates that Emotion uses Node version 12. After running `nvm use` in the repository I was able to successfully `yarn install` and get past a similar `sharp` compliation error.

**Comment by srmagura** [comment]
This is an issue with M1 Macs and older versions of the `sharp` package. To fix, see the first bullet point under "Installation" in [CONTRIBUTING.md](https://github.com/emotion-js/emotion/blob/2d3d7dd1d8b0de9dbfd50e42ee716ac00fb70ecd/CONTRIBUTING.md).

This will be resolved once #2571 is merged.
